<ui version="4.0" >
 <class>ContentEditor_BASE</class>
 <widget class="QDialog" name="ContentEditor_BASE" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Content Editor</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_ViewContentAs" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_ViewAsType" >
        <property name="text" >
         <string>View content as:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_ContentType" >
        <item>
         <property name="text" >
          <string>Text</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Hex</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit" >
     <property name="cursor" stdset="0" >
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="lineWrapMode" >
      <enum>QTextEdit::WidgetWidth</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pushButton_Revert" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Export" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Browse" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>comboBox_ContentType</tabstop>
  <tabstop>pushButton_Browse</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
  <tabstop>pushButton_OK</tabstop>
  <tabstop>pushButton_Revert</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>ContentEditor_BASE</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>354</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel" >
     <x>197</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>ContentEditor_BASE</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>273</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel" >
     <x>197</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
